DevOps/frankfurter/resources/interfaces.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+"""
+interfaces -- Create and copy a custom network interface
+"""
+
+import os
+import paramiko
+from scp import SCPClient
+
+HOST = '192.168.7.2'
+USERNAME, PASSWORD = 'ubuntu', 'temppwd'
+TMP_FILE_PATH = '/tmp/interfaces'
+
+
+def read_team_number(prompt='Enter the team number: '):
+    while True:
+        try:
+            return int(input(prompt))
+        except ValueError:
+            print('Not a number.')
+
+
+def build_custom_config(team_number):
+    with open('interfaces') as network_config_file:
+        for line in network_config_file:
+            if line.startswith('iface ra0'):
+                line = line.replace('dhcp', 'static')
+            yield line[:-1]  # Take off the newline
+            if 'wpa-psk' in line:
+                indent = ' '*4
+                yield indent + 'address 192.168.128.{0}'.format(team_number)
+                yield indent + 'netmask 255.255.255.0'
+                yield indent + 'network 192.168.128.0'
+                yield indent + 'gateway 192.168.128.1'
+
+
+def write_tmp_file(lines):
```

Comment:
for future reference, consider:
https://docs.python.org/3.5/library/tempfile.html